src/features/events/utils/eventUtils.ts

```diff
@@ -0,0 +1,16 @@
+import theme from 'theme';
+
+export function getParticipantsStatusColor(
```

Comment:
I like this, but it's a copy from the logic in other places. That's fine though, we will just have to refactor it later.